MPChartLib/src/main/java/com/github/mikephil/charting/components/YAxis.java

```diff
@@ -357,12 +378,54 @@ public boolean needsOffset() {
             return false;
     }
 
+    /**
+     * Sets min value restriction for autoscale
+     */
+    public void setAutoScaleMinRestriction(float restrictionValue) {
```

Comment:
remove these two setters: `setAutoScaleMinRestriction()` and `setAutoScaleMaxRestriction()`